src/api/Feed.js

```diff
@@ -1128,6 +1277,40 @@ class Feed extends Base {
         }
     };
 
+    /**
+     * Deletes a reply from the cache
+     *
+     * @param {string} id - The id of the feed item to be deleted
+     * @param {string} parentId - The id of the parent feed item
+     * @param {Function} successCallback - function to be called after the delete
+     */
+    deleteReplyItem = (
+        id: string,
+        parentId: string,
+        successCallback: (id: string, parentId: string) => void = noop,
+    ) => {
+        const cachedItems = this.getCachedItems(this.file.id);
+        if (cachedItems) {
```

Comment:
Minor, but it seems like we could remove some nesting here with a change to the line above:

```
const cachedItems = this.getCachedItems(this.file.id) || [];

Reply:
The default `cachedItems` actually needs to be `{ items: [], errors: [] }`, but I have made he change.